frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -271,7 +278,7 @@ const composeAndPushNotificationMessages = (
 const handleSubmit = async () => {
     const successForestClientIdList: string[] = [];
     const warningForestClientIdList: string[] = [];
-    const errorForestClientIdList: string[] = [];
+    const errorForestClientIdList = {errorMsg: '', forestClienNumber: [] as string[]};
```

Comment:
Strange to see the "**error**..." variable "type" is different than the "success/warning" ones.
So this variable was specific to "**idList**" only, now you change its definition, the variable should reflect that and not leaving it hanging for other developer to get confused.